code/learning_material/MessageList.js

```diff
@@ -0,0 +1,69 @@
+// This is all the material from the Monday lecture
```

Comment:
I also did the MessageList without being able to actually use it in my final project. 
It ended up looking totally different just like yours.